### ISSUE TITLE
Find a place store and share data

### ISSUE THREAD

**Issue by ofrancon** [Project-Resilience/mvp#14]
The notebooks in `mvp/data/` download public data, process it and persist it locally. It should be possible for Project Resilience contributors to directly download and use the processed data, without having to re-run the notebooks that generate it.

One solution could be to use some ITU AWS account and have an S3 bucket there.

**Comment by jacobbieker** [comment]
Huggingface could be another one? Easy access and free hosting. It's where we put all our public data

**Comment by ofrancon** [comment]
Yes that could work too.

**Comment by jacobbieker** [comment]
The merged, aggregated dataset from #17 I've put on [HuggingFace](https://huggingface.co/datasets/jacobbieker/project-resilience), and it can be accessed here:

```python
import xarray as xr

f = "zip:///::https://huggingface.co/datasets/jacobbieker/project-resilience/resolve/main/merged_aggregated_dataset.zarr.zip"
dataset = xr.open_dataset(f, engine='zarr', chunks={})
print(dataset)
```
